JapaneseFilter/sections/specific.txt

```diff
@@ -25,7 +25,7 @@ ebitsu.net,news.2chblog.jp,chomagireko.blog.jp,tarosoku.com,0taku.livedoor.biz,k
 ! `##.adsense`
 news.biglobe.ne.jp,4ka.net,natsukimemo.com,travelbook.co.jp,cat.sc,battle-one.com,otonasalone.jp##.adsense
 ! `##.banner`
-climbing-net.com,tripnote.jp,allreviews.jp,smart-flash.jp,news.fresheye.com,pakutaso.com,gaydebulog.com,qoly.jp,ascii2d.net,hoikuen-ranking.com,jp.msn.com,dengekionline.com,wiki.seesaa.jp##.banner
+toto.rakuten.co.jp,limbing-net.com,tripnote.jp,allreviews.jp,smart-flash.jp,news.fresheye.com,pakutaso.com,gaydebulog.com,qoly.jp,ascii2d.net,hoikuen-ranking.com,jp.msn.com,dengekionline.com,wiki.seesaa.jp##.banner
```

Comment:
Move to self-promo. BTW `climbing-net.com` is mistakenly broken.

Reply:
Also add `toto.rakuten.co.jp#?#.login-section2 > div.login-none-inner:has(> div.campaign)` to self-promo.

Reply:
OK/ done.